go.mod

```diff
@@ -1,9 +1,11 @@
 module github.com/anchore/grype
 
-go 1.21.1
+go 1.22.0
+
+toolchain go1.22.4
```

Comment:
It would be great if we could omit the toolchain directive. The syft go mod only has `go 1.22.0`

Reply:
Ahh yea - new laptop with up to date toolchain installed so it does this automagically on go mod tidy =(